cpp/src/join/mixed_join.cu

```diff
@@ -109,9 +109,9 @@ mixed_join(
   // If evaluating the expression may produce null outputs we create a nullable
   // output column and follow the null-supporting expression evaluation code
   // path.
-  auto const has_nulls =
+  auto const has_nulls = cudf::nullate::DYNAMIC{
```

Comment:
Please make sure if `has_nested_nulls` is actually what we want here.

Reply:
mixed joins will not be supporting nested types